app/views/collections/_document_list.html.erb

```diff
@@ -1,3 +1,5 @@
+<% if documents.empty? %> <h3><%= t('sufia.collection.no_works') %></h3><br>
```

Comment:
No, sorry. This is what I had in mind:

```ruby
<% if documents.empty? %>
<h3><%= t('sufia.collection.no_works') %></h3>
<% end %>
```

Not an HTML line break but a CRLF.